src/machines/commerce-machine.ts

```diff
@@ -38,6 +38,17 @@ const extractAppliedDefaultCoupon = (
     }
   }
 
+  // if we have an appliedCoupon that isn't region restricted and not default
+  // we want to honor it
+  if (appliedCoupon && !appliedCoupon.coupon_region_restricted) {
+    return {
+      couponToApply: {
+        couponCode: appliedCoupon.coupon_code,
+        couponType: 'special',
+      },
+    }
+  }
```

Comment:
non-default non-ppp applied coupon is still valid and is being PRESENTED as the price but not carrying through to Stripe